### PR TITLE
Modern theme sort

### PR DESCRIPTION
**Pull request by Blackbaud-ToddRoberts**

### CODE REVIEW

File: src/app/public/modules/sort/sort-menu-heading.component.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<div
+  class="sky-sort-menu-heading"
+  [skyThemeClass]= "{
+    'sky-subsection-heading': 'default',
+    'sky-font-heading-4': 'modern'
+  }">
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/sort/sort-menu-heading.component.html:6]: Closing `>` needs to be on the next line.

**Review by Blackbaud-AlexKingman — COMMENTED** [review]
Looks good, but I think you'll need to tweak the unit tests to be aware of your new changes.

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-lists?_sp.spa=pr-modern-theme-sort](https://developer.blackbaud.com/skyux-lists?_sp.spa=pr-modern-theme-sort)

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-lists?_sp.spa=pr-modern-theme-sort](https://developer.blackbaud.com/skyux-lists?_sp.spa=pr-modern-theme-sort)

**Review by Blackbaud-MattGregg — APPROVED** [review]
Looks good to me

**Review by Blackbaud-AlexKingman — APPROVED** [review]

**Pull request merged**